opm/parser/eclipse/Parser/Parser.cpp

```diff
@@ -432,7 +433,7 @@ std::shared_ptr< RawKeyword > createRawKeyword( const string_view& kw, ParserSta
 
     std::string msg = "Expected the kewyord: " + sizeKeyword.first
                     + " to infer the number of records in: " + keywordString;
-
+    parserState.deck->getMessageContainer().error(msg);
```

Comment:
Yes, that's the reason I opened an issue, see https://github.com/OPM/opm-parser/issues/790